src/legacy/core_plugins/kibana/public/context/query_parameters/actions.js

```diff
@@ -73,7 +73,8 @@ export function QueryParameterActionsProvider(indexPatterns, Private) {
 
   const addFilter = (state) => async (field, values, operation) => {
     const indexPatternId = state.queryParameters.indexPatternId;
-    filterManager.add(field, values, operation, indexPatternId);
+    const newFilters = filterGen.generate(field, values, operation, indexPatternId);
```

Comment:
why were we using filterManager before and we are using queryFilter now ?

Reply:
Old `filterManager` was renamed to `filterGen` as the naming was confusing.
They will be merged in a subsequent PR.